### PR TITLE
Update rc with recent API changes

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/pages/resultsView/enrichments/EnrichmentsUtil.tsx [hunk 1]
```diff
@@ -81,10 +81,10 @@ export function getAlterationRowData(alterationEnrichments: AlterationEnrichment
...
             entrezGeneId: alterationEnrichment.entrezGeneId,
             cytoband: alterationEnrichment.cytoband, 
-            alteredCount: alterationEnrichment.alteredCount,
-            alteredPercentage: alterationEnrichment.alteredCount / totalAltered * 100,
-            unalteredCount: alterationEnrichment.unalteredCount,
-            unalteredPercentage: alterationEnrichment.unalteredCount / totalUnaltered * 100,
+            alteredCount: alterationEnrichment.set1CountSummary.alteredCount,
+            alteredPercentage: alterationEnrichment.set1CountSummary.alteredCount / totalAltered * 100,
+            unalteredCount: alterationEnrichment.set2CountSummary.alteredCount,
+            unalteredPercentage: alterationEnrichment.set2CountSummary.alteredCount / totalUnaltered * 100,
```
Comment (adamabeshouse) [inline src/pages/resultsView/enrichments/EnrichmentsUtil.tsx]: @onursumer @zhx828 @averyniceday should the denominator be ..CountSummary.profiledCount? or total count as it is now?

Comment (zhx828) [reply src/pages/resultsView/enrichments/EnrichmentsUtil.tsx]: profiledCount sounds right to me. This is similar to what we have in study view about sequenced vs all?

Comment (jjgao) [reply src/pages/resultsView/enrichments/EnrichmentsUtil.tsx]: I think it should be profiledCount. Note that different genes may have different profiledCount. We should really take the count of samples/patients that are profiled for both genes and it should probably be returned in the alterationEnrichment result.

How does the backend do the counting?

Comment (adamabeshouse) [reply src/pages/resultsView/enrichments/EnrichmentsUtil.tsx]: @averyniceday can you answer this?

Comment (averyniceday) [reply src/pages/resultsView/enrichments/EnrichmentsUtil.tsx]: @ao508 can you comment on this?

**Pull request merged**